From 8ac494592bb738e1b7adf23bb0dcbc8bf1d4e22f Mon Sep 17 00:00:00 2001
From: German Kirsanov <germankirsanov2011@yandex.ru>
Date: Sat, 26 Aug 2017 15:15:07 +0500
Subject: [PATCH] Fix build

---
 MiuiHome/apktool.yml | 1 +
 1 file changed, 1 insertion(+)

--- a/MiuiHome/apktool.yml
+++ b/MiuiHome/apktool.yml
@@ -14,6 +14,7 @@ usesFramework:
   ids:
   - 1
   - 16
+  - 27
 version: 2.2.1-93a51c-SNAPSHOT
 versionInfo:
   versionCode: '10000'
-- 
2.14.0